app/src/constants.js

```diff
@@ -1,10 +1,11 @@
 const Constants = {
-    KEY_FIRST_RUN_EMAIL: "firstRunEmail",
-    KEY_SHOULD_VERIFY_EMAIL: "shouldVerifyEmail",
+  KEY_FIRST_RUN_EMAIL: "firstRunEmail",
+  KEY_SHOULD_VERIFY_EMAIL: "shouldVerifyEmail",
 
-    SETTING_ALPHA_UNDERSTANDS_RISKS: "alphaUnderstandRisks",
+  SETTING_ALPHA_UNDERSTANDS_RISKS: "alphaUnderstandRisks",
 
-    ACTION_FIRST_RUN_PAGE_CHANGED: "FIRST_RUN_PAGE_CHANGED",
+  ACTION_DELETE_COMPLETED_BLOBS: "DELETE_COMPLETED_BLOBS",
```

Comment:
Why are some still mixed formatted? (camel vs underscores)

Reply:
Following convention with the constants defined in `lbry-desktop` and `lbry-redux`. The settings and keys are mixed case. The redux actions are uppercase with underscores.
https://github.com/lbryio/lbry-redux/blob/master/src/constants/action_types.js
https://github.com/lbryio/lbry-redux/blob/master/src/constants/settings.js